submissions/adeolaFafemi/phpDatabases/customerOrdersList.php

```diff
@@ -0,0 +1,55 @@
+<?php
+include('connection.php');
+
+if (isset($_GET['id'])){
+    $id = $_GET['id'];
+}
+$sql = "SELECT products.name, order_items.unit_price, order_items.quantity, order_items.total_amount,
```

Comment:
The way this query is placed here, anytime I reload this page, it will always be executed even when cookie is set. Kindly figure out a way to prevent this from happening such that the query only gets executed when I want it too